### ISSUE TITLE
`Airbrake::Error: :project_id is required` is thrown for ignored environment 

### ISSUE THREAD

**Issue by ivanovv** [airbrake/airbrake#931]
I had top update Airbrake gem version because of frequent deprecation warning in the logs:
```
~/.rvm/rubies/ruby-2.5.1/lib/ruby/gems/2.5.0/gems/airbrake-8.1.4/lib/airbrake/rack/middleware.rb:35: warning: Airbrake#notifiers is deprecated. It will be removed from airbrake-ruby v4 altogether.
```
Once I did that and ended up with this in `Gemfile.lock`
```
    airbrake (8.2.1)
      airbrake-ruby (~> 4.1)
    airbrake-ruby (4.1.0)
      rbtree3 (~> 0.5)
```

CI tests started to fail with the following message
```
bundle exec rake db:structure:load
rake aborted!
Airbrake::Error: :project_id is required
/home/circleci/boomrdashboard/vendor/bundle/ruby/2.5.0/gems/airbrake-ruby-4.1.0/lib/airbrake-ruby.rb:105:in `configure'
/home/circleci/boomrdashboard/config/initializers/airbrake.rb:10:in `<top (required)>'
/home/circleci/boomrdashboard/vendor/bundle/ruby/2.5.0/gems/activesupport-5.0.7/lib/active_support/dependencies.rb:287:in `load'
```

Airbrake config:
```
Airbrake.configure do |c|
  c.project_id = ENV['AIRBRAKE_PROJECT_ID']
  c.project_key = ENV['AIRBRAKE_API_KEY']
  c.root_directory = Rails.root
  c.logger = Rails.logger
  c.environment = Rails.env
  c.ignore_environments = %w[development test]
  c.blacklist_keys = [/password/i, /authorization/i]
  c.route_stats = true
end
```

`AIRBRAKE_PROJECT_ID` & `AIRBRAKE_API_KEY` env vars are not set for `test` env

**Comment by rwojnarowski** [comment]
In previous versions for me it was working setting project_id env in Test Environment to empty string:
```
AIRBRAKE_PROJECT_ID = ""
```
after latest updates, i have to update it to not empty string:
```
AIRBRAKE_PROJECT_ID = "anything"
```

**Comment by alex-mohemian** [comment]
Same here with 8.3.2

**Comment by gsar** [comment]
seeing the same thing with our app on 8.3.2.  culprit seems to be this change: https://github.com/airbrake/airbrake-ruby/commit/4018bee51f5819adc917a2751a0fac5cf6695f24

**Comment by kyrylo** [comment]
Apologies for this, it was an oversight.
Should be fixed in airbrake-ruby v4.2.1

```
bundle update airbrake-ruby
```
